keras/callbacks.py

```diff
@@ -2350,12 +2349,13 @@ class TensorBoard(Callback, version_utils.TensorBoardVersionSelector):
         write_steps_per_second: whether to log the training steps per second
           into Tensorboard. This supports both epoch and batch frequency
           logging.
-        update_freq: **disabled**
-
-          Warning: Batch-level summary writing using `update_freq` is
-          currently unsupported. A suggested workaround is shown in the
-          [TensorBoard Scalars tutorial](https://www.tensorflow.org/tensorboard/scalars_and_keras#batch-level_logging). # pylint: disable=protected-access
-
+        update_freq: `'batch'` or `'epoch'` or integer. When using `'batch'`,
+          writes the losses and metrics to TensorBoard after each batch. The
+          same applies for `'epoch'`. If using an integer, let's say `1000`, the
+          callback will write the metrics and losses to TensorBoard every 1000
+          batches. Note that writing too frequently to TensorBoard can slow down
+          your training. May not work when doing distributed training, as
+          currently only a subset of `tf.distribute.Strategy`s are supported.
```

Comment:
Can we make it clear that this is not supported in ParameterServerStrategy, and may have performance implication when using `tf.distribute`?